lib/mayaUsd/CMakeLists.txt

```diff
@@ -62,14 +62,7 @@ if(DEFINED UFE_PREVIEW_VERSION_NUM)
     )
 endif()
 
-if (MAYA_API_VERSION VERSION_GREATER_EQUAL 20220100 AND MAYA_API_VERSION VERSION_LESS 20230000)
-    target_compile_definitions(${PROJECT_NAME}
-        PRIVATE
-            HAS_DEFAULT_MATERIAL_SUPPORT_API=1
-    )
-endif()
-
-if (MAYA_API_VERSION VERSION_GREATER_EQUAL 20230000 AND UFE_VERSION VERSION_GREATER_EQUAL 2.0.3)
+if (MAYA_DEFAULT_MATERIAL_API EQUAL 1)
```

Comment:
Some recent build breaks have cast doubt on these combinations of versions. FindMaya.cmake now makes sure the API is correctly detected.